src/routes/controllers/users.controller.js

```diff
@@ -189,3 +189,53 @@ export async function getUsersBookmarks(req, res) {
     return errorResponse(res, 500, error.message);
   }
 }
+
+/**
+ *
+ * @param {Object} req express request
+ * @param {Object} res express response
+ * @returns {Array} user likes
+ */
+export async function getAllArticlesLikedByUser(req, res) {
+  const { user } = req;
+  const { type } = req.params;
```

Comment:
I was thinking we should get the `type` from req query instead of params that way we can just call `/likes` to fetch both articles and comments like and `/like?type=[comment|article]` to either return articles or comments.

Reply:
I have figured out how to implement this and I have implemented it.